zip-####.rst

```diff
@@ -50,37 +51,82 @@ Code (Title 26 of the U.S. Code). [#section501c3]_
+organization incorporated and based in the Cayman Islands.
+<https://www.financialprivacyfoundation.org/>
+
+"Qedit" refers to the company founded in 2016 by a world-class team of 
+accomplished tech entrepreneurs, researchers, and developers; 
+QEDIT has emerged as a global leader in the field of Zero-Knowledge Proofs.
+<https://qed-it.com/about-us/>
+
+"ZecHub" refers to the team of content creators who have supported Zcash
+through a series of ZCG approved grants.
+<https://forum.zcashcommunity.com/t/zechub-an-education-hub-for-zcash-2024-continued/47947>
+
 The terms "Testnet" and "Mainnet" are to be interpreted as described in
 section 3.12 of the Zcash Protocol Specification [#protocol-networks]_.
 
+The term "ZSA" is to be interpreted as "Zcash Shielded Assets" the protocol
+feature enhancement (and subsequent application layer, legal, maintenance 
+efforts, et al) on-going by Qedit, et al.
+<https://forum.zcashcommunity.com/t/grant-update-zcash-shielded-assets-monthly-updates/41153>
+
 
 Abstract
 ========
 
 This proposal describes a structure for the Zcash Development Fund, to be
-enacted in Network Upgrade 4 and last for 4 years. This Dev Fund would consist
-of 20% of the block subsidies, split into 3 slices:
+enacted in Network Upgrade 6 and last for 4 years. This Dev Fund would consist
+of 15% of the block subsidies, as the following allocations:
+
+* 5% for the Zcash Foundation (for internal work and grants);
+* 4% for the Bootstrap Project (the parent of the Electric Coin Company) for the 1st year only; 
+and contingent upon an ecosystem mandated "non-direct funding model" for the future;
+* 4% for Zcash Community Grants for continuation of their current activities as-is;
+* 2% for Qedit in support of their ZSA activities, and other Zcash protocol support;
 
-* 35% for the Bootstrap Project (the parent of the Electric Coin Company);
-* 25% for Zcash Foundation (for internal work and grants);
-* 40% for additional "Major Grants" for large-scale long-term projects
-  (administered by the Zcash Foundation, with extra community input and
-  scrutiny).
+Following the first year, when ECC will no longer receive their 4% allocation, 
+those block subsidies will be distributed as 1% to ZCG, 1% to ZecHub, 1% to FPF,
+and 1% to ZF.
 
 Governance and accountability are based on existing entities and legal mechanisms,
-and increasingly decentralized governance is encouraged.
+and increasingly decentralized governance is encouraged.  This proposal mandates 
+the ecosystem to design and deploy a "non-direct funding model" as generally
+recommended in Josh Swihart's proposal.
+<https://forum.zcashcommunity.com/t/zcash-funding-bloc-a-dev-fund-proposal-from-josh-at-ecc/47187>
+
+Upon creation/ activation of a "non-direct funding model" this ZIP should be 
+reconsider by the Zcash ecosystem, to determine if it's ongoing direct block subsidies are 
+a preference. Discussions/ solications/ sentiment gathering from the Zcash
```

Comment:
```suggestion
reconsidered by the Zcash ecosystem, to determine if its ongoing direct block subsidies 
should continuence. Discussions/ solications/ sentiment gathering from the Zcash
```

Reply:
"continuence" doesn't feel like the correct wording either,  but I agree that the end of that statement can be improved.

"its ongoing direct block subsidies are preferred for continuation."  _(would that be satisfactory?)_